src/Routing/TermRouteSubscriber.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\helfi_platform_config\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+use Drupal\Core\Routing\RoutingEvents;
+
+/**
+ * Class TermRouteSubscriber.
```

Comment:
`Class TermRouteSubscriber.` and `@package Drupal\helfi_platform_config\Routing` comments can be removed since they provide nothing of value.

Reply:
Good point, removed.